dora/tests/src/test/java/alluxio/client/cli/fs/command/ChecksumCommandIntegrationTest.java

```diff
@@ -36,7 +36,7 @@ public final class ChecksumCommandIntegrationTest extends AbstractFileSystemShel
    */
   @Test
   public void checksum() throws Exception {
-    FileSystemTestUtils.createByteFile(sFileSystem, "/testFile", WritePType.MUST_CACHE, 10);
+    FileSystemTestUtils.createByteFile(sFileSystem, "/testFile", WritePType.CACHE_THROUGH, 10);
```

Comment:
For this kind of manually set write types, what do you think about declaring a shared var?
```
class ChecksumCommandIntegrationTest {
  public static final WritePType DEFAULT_WRITE_TYPE = WritePType.CACHE_THROUGH;

  // keep using this constant throughout your test
}
```

You don't save any code in this PR, but you may save a lot in later PRs (if you tweak the default write type).
All these test class files may share the same `DEFAULT_WRITE_TYPE` if you find a good place to declare this constant.

Reply:
@jiacheliu3 I'm not sure if we expect a default value for the different unit tests. For the general default value, we only need to set `DEFAULT_WRITE_TYPE` in the cluster resouce configuration in some based test class. 
Except that, I saw the most cases is it wants to excute a specific opration with a specific write type. we can not easily replace them with a `DEFAULT_WRITE_TYPE` in general.